### ISSUE TITLE
Editorial improvements

### ISSUE THREAD

**Issue by strugee** [w3c-social/activitypump#142]
Minor editorial problems (I've been careful to list only those that I feel confident won't affect implementations so you don't have to worry about this minor stuff before CR):
- Section 2 Conformance contains (almost) the same text twice - from looking at the source, seems like reSpec automatically inserts this and you can remove the text in the HTML
- 2.2 Candidate Recommendation Exit Criteria, third bullet point of the "considered a feature" list: "handling side client to server interaction effects appropriately" is kind of awkward and hard to parse
- Most (all?) `Content-Type` AS2.0 namespace references in the document have a trailing `#`, which seems incorrect. I.e. the document gives a `Content-Type` of `application/ld+json; profile="https://www.w3.org/ns/activitystreams#"`, but it seems like it should be `application/ld+json; profile="https://www.w3.org/ns/activitystreams"`
- Missing a trailing `.` at the end of the second paragraph of 3.2 Methods on objects
- 4.1 Actor objects, in the descriptions of `inbox` and `outbox`: s/comprising/comprised/
- 5.2 Following Collection (maybe other places in the document): possible inconsistent formatting of the collection name: "Following" vs. "`following`"
- 5.3 Likes Collection: s/`following`/`likes`/ (presumably), although with that simple replacement you'd end up with the same formatting inconsistency (can't tell if it's intentional) as in 5.2 Following Collection
- 6. Binary Data, Example 5: the `https://example.com/~erik/public` addressing is pretty confusing. Is this supposed to be the special Public Collection address? If so seems like a typo
- 7. Client to Server Interactions, paragraph 4: "the Activity" is unclear - presumably this is any Activity that the client processes, but as it is this is grammatically incorrect as it's not clear which Activity is being referred to
- 7. Client to Server Interactions, Example 8: I'm 99% sure the HTTP request line is incorrect. It should be: `HTTP/1.1 /outbox/ POST`
- 7.1 Create Activity, paragraph 1: "is used to when" should (AFAICT) be "is used when"
- 7.1 Create Activity, paragraph 1: "created" (the last word in the paragraph) is a little unclear - perhaps "created within the system"? Not sure if that's actually better
- 7.1 Create Activity, paragraph 3: "Note that it is acceptable for the `object`'s addressing may be changed later..." doesn't make sense, probably s/may/to/?
- 7.1.1 Object creation without a Create Activity: "it is possible to create a new object without a surrounding activity" could be clearer - perhaps something along the lines of, "it is possible to submit an object for creation without a surrounding activity."
- 7.1.1 Object creation without a Create Activity, example 11: invalid JSON: missing comma after `"type": "Create"`
- 7.8 Block Activity - "The server SHOULD prevent the user from interacting..." should be changed to "The server SHOULD prevent the _blocked_ user from interacting..." (emphasis added)
- 8.1 Targeting: the list contains items that aren't continuations of the sentence that precedes them. E.g. "Thus, the targets for delivery of an activity may be primary targeting" doesn't quite make sense. I suggest maybe adding the names in parenthesis at the end, i.e. "the values of the `to`, `cc`, or `bcc` fields (likely to include the followers collection) (**primary targeting**)
- 8.1 Targeting, paragraph 3: "Servers MUST also exclude actors from the list which are the same as the `actor` of the activity being notified about" makes sense, but is _really_ hard to parse. I suggest adding another sentence after that one to clarify, e.g.: "That is, users shouldn't have their own activities delivered to themselves."
- 8.2 Delivery, paragraph 1 has far too many commas. Suggest keeping the first, removing the next two, and adding the word "subsequent" before the the second "delivery"
- 8.2 Delivery, paragraph 2: s/OrderedCollection/`OrderedCollection`/
- 8.2.1 Outbox Delivery, paragraph 1: it'd be nice if the word "target" was a link to section 8.1 Targeting
- 8.2.1 Outbox Delivery, in the list: remove the comma after "individuals"
- 9.2 Linked Data Signatures and HTTP Signatures: s/json-ld/JSON-LD/
- 9.2 Linked Data Signatures and HTTP Signatures, paragraph 2: it's not obvious how one might "link the public key to the actor's profile object"; this would really benefit from a link to 4.1 Actor objects where properties specifically for this are mentioned
- 9.2.1 Client to Origin Server Authentication: the second comma is unnecessary as is the "then" after "and" in the same spot
- 9.2.1 Client to Origin Server Authentication: "subsequently signing HTTP requests made with this key" is grammatically ambiguous; I suggest "subsequently using this key to sign any HTTP requests made."
- C. Security and Privacy Review, question 1: "ActivityPub does not mandate that any personally identifiable information to operate" is clearly a typo

**Comment by ayleph** [comment]
> 2.2 Candidate Recommendation Exit Criteria, third bullet point of the "considered a feature" list: "handling side client to server interaction effects appropriately" is kind of awkward and hard to parse

I noticed the same thing. It looks like a word got out of place in copy/paste. The fifth bullet point is worded as "handling server to server side effects appropriately," which makes me think the third bullet point was intended to say "handling client to server interaction side effects appropriately."

**Comment by rhiaro** [comment]
Thank you SO much for this @strugee and I can only apologise for how long it's taken to get through your list. I took all of your suggestions into account that were still applicable following changes since you posted. Really appreciate the time you took to read through.

**Comment by strugee** [comment]
@rhiaro happy to help! You don't need to apologize; I'm sure everyone in this WG is super busy. I know I am.

**Comment by strugee** [comment]
If the spec has changed significantly I'd be down to read through it again, if it'd be helpful. I seem to have a knack for finding editorial issues :)
